inputfiles/addedTypes.json

```diff
@@ -1175,7 +1175,10 @@
                         }
                     }
                 },
-                "no-interface-object": "1"
+                "no-interface-object": "1",
+                "includes": [
+                    "DocumentOrShadowRoot"
```

Comment:
We don't need this manual `includes` field as the DOM spec already takes care of this.